ledger-api/grpc-definitions/com/daml/ledger/api/v1/transaction_service.proto

```diff
@@ -16,34 +16,73 @@ option java_package = "com.daml.ledger.api.v1";
 option csharp_namespace = "Com.Daml.Ledger.Api.V1";
 
 // Allows clients to read transactions from the ledger.
+// ``INVALID_ARGUMENT`` gRPC error will be returned in case of malformed payloads and missing required fields.
+// ``NOT_FOUND`` will be returned in case a request specifies a ledger id that is not valid.
+//
+// When the participant is run in mode requiring authentication, all the calls
+// in this interface will respond with ``UNAUTHENTICATED``, if the caller fails
+// to provide a valid access token, and will respond with ``PERMISSION_DENIED``, if
+// the claims in the token are insufficient to perform a given operation.
```

Comment:
We don't need this paragraph anymore, right?

Reply:
Good catch!

Reply:
Fixed.